docs/_static/theme_overrides.css

```diff
@@ -1,15 +0,0 @@
-
```

Comment:
This is actually useful for correctly displaying overflow in the table cells, any reason why it's deleted?

Reply:
It had issues building on RTD servers (but worked locally) so got commented out.. found a way to add it in #2599